lms/djangoapps/verify_student/tasks.py

```diff
@@ -114,11 +114,12 @@ def send_request_to_ss_for_user(self, user_verification_id, copy_id_photo_from):
     log.info('=>New Verification Task Received %r', user_verification.user.username)
     try:
         headers, body = user_verification.create_request(copy_id_photo_from)
+        # checkout PROD-1395 for detail why we are adding system certificate paths for verification.
         response = requests.post(
             settings.VERIFY_STUDENT["SOFTWARE_SECURE"]["API_URL"],
             headers=headers,
             data=simplejson.dumps(body, indent=2, sort_keys=True, ensure_ascii=False).encode('utf-8'),
-            verify=False
+            verify='/etc/ssl/certs/ca-certificates.crt'
```

Comment:
could we remove this argument altogether and test on the sandbox whether that would make any difference? (I'm assuming there isn't time sensitivity on pushing this forward; if I'm wrong feel free to disregard.)

Reply:
@matthugs we did earlier removed that argument and we got verification error and we did revert the PR. To fix that certification error we are adding specific path and there isn't any sensitivity issue we can pass the path of certificate to requests.

Reply:
@adeelehsan Can we make our system's CA bundle as request's default bundle instead of hardcoding the path? I'm not sure but just for the safe side if we can are able to that.

Reply:
Check this out https://github.com/psf/requests/blob/bebf5250b0df1b922ff534e987a4481cc4dde99d/docs/user/advanced.rst#ssl-cert-verification, there is an environment variable `REQUESTS_CA_BUNDLE` to specify trusted certificates.